packages/qwik/src/optimizer/src/plugins/vite.ts

```diff
@@ -105,6 +105,22 @@ export function qwikVite(qwikViteOpts: QwikVitePluginOptions = {}): any {
 
       qwikPlugin.log(`vite config(), command: ${viteCommand}, env.mode: ${viteEnv.mode}`);
 
+      if (sys.env === 'node' && !qwikViteOpts.entryStrategy) {
+        const fs: typeof import('fs') = await sys.dynamicImport('node:fs');
```

Comment:
Why not do a regular import? Why dynamic?

Reply:
I handled the import the same way as the rest of the code.
eg. [few lines after my code](https://github.com/BuilderIO/qwik/blob/a4b0ca9b1815c4d8c6fec4c7d02f62bf7427da51/packages/qwik/src/optimizer/src/plugins/vite.ts#L181)